CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Fixed connectors not showing on operator's "All connectors" list ([#119](https://github.com/sovity/authority-portal/issues/119))
 - Changed Access Token Lifespan to be one minute ([#111](https://github.com/sovity/authority-portal/issues/111))
 - Fixed an issue with connector URLs not being validated correctly in the UI ([#115](https://github.com/sovity/authority-portal/issues/115))
+- Fixed "Up since ..." in the dashboard view page ([#62](https://github.com/sovity/authority-portal/issues/62))
```

Comment:
Fix wording here: "on the dashboard page"

Or adjust the wording completely to something along the lines of "Dashboard: Changed the component uptime duration to be shown next to the status instead of showing it in a tooltip"